changelogs/unreleased/4353-jxun

```diff
@@ -0,0 +1 @@
+Modify generated from schedule's backup name timestamp to UTC timestamp.
```

Comment:
Apostrophe possessive in awkward in English for multi-word phrases. Perhaps
```suggestion
Modify the timestamp in the name of a backup generated from schedule to use UTC.
```

Reply:
Thanks. I'll reword it.